examples/quickstart/README.md

```diff
@@ -5,6 +5,6 @@ This page contains a list of quickstart notebooks written with Open Bandit Pipel
 - [`obd.ipynb`](./obd.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/obd.ipynb): a quickstart guide of using Open Bandit Dataset and Pipeline to conduct some OPE experiments.
 - [`synthetic.ipynb`](./synthetic.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/synthetic.ipynb): a quickstart guide to implement the standard off-policy learning, OPE, and the evaluation of OPE on synthetic bandit data with Open Bandit Pipeline.
 - [`multiclass.ipynb`](./multiclass.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/multiclass.ipynb): a quickstart guide to handle multi-class classification data as logged bandit data for the standard off-policy learning, OPE, and the evaluation of OPE with Open Bandit Pipeline.
-- [`online.ipynb`](./online.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/online.ipynb): a quickstart guide to implement OPE and the evaluation of OPE for online bandit algorithms with Open Bandit Pipeline.
+- [`online.ipynb`](./replay.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/online.ipynb): a quickstart guide to implement OPE and the evaluation of OPE for online bandit algorithms with Open Bandit Pipeline.
```

Comment:
```suggestion
- [`reply.ipynb`](./replay.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/st-tech/zr-obp/blob/master/examples/quickstart/online.ipynb): a quickstart guide to implement OPE and the evaluation of OPE for online bandit algorithms with Open Bandit Pipeline.
```

Reply:
Hey! Not sure what you mean with this suggestion? You want to keep this as `replay`?

Reply:
@Braamling Since you renamed `online.ipynb` -> `reply.ipynb`, I think we should also update this README file accordingly, because there's no file named `online.ipynb` now. Sorry if I misunderstand something...

Reply:
Got it! Github didn't show the diff in the right place, so it got me confused. It should be change now :) 

I think we should be there now! Let me know if you think this is ready to merge.